torch/nn/modules/module.py

```diff
@@ -419,8 +424,13 @@ def to(self, *args, **kwargs):
                 raise TypeError('nn.Module.to only accepts floating point '
                                 'dtypes, but got desired dtype={}'.format(dtype))
 
+        convert_to_format = kwargs.get('memory_format', None)
+
         def convert(t):
-            return t.to(device, dtype if t.is_floating_point() else None, non_blocking)
+            result = t
+            if convert_to_format is not None and t.dim() == 4:
+                result = t.contiguous(memory_format=convert_to_format)
```

Comment:
if you can't call `to(memory_format=...)` for now that seems fine, just comment it.

Reply:
done